jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ImageToJsonTranslator.java

```diff
@@ -141,6 +141,9 @@ public Blob getContainerConfigurationBlob() {
     // Sets the working directory.
     template.setContainerWorkingDir(image.getWorkingDirectory());
 
+    // Sets the user.
+    template.setContainerUser(image.getUser());
```

Comment:
Should there be a counterpart for `JsonToImageTranslator`?

Reply:
Yes, for future use